src/cljs/nr/stats.cljs

```diff
@@ -13,6 +13,15 @@
             [nr.ws :as ws]
             [reagent.core :as r]))
 
+(def state (r/atom {:games nil}))
+
+(defn- fetch-game-history []
+  (go (let [{:keys [status json]} (<! (GET "/profile/history"))
+            games (map #(assoc % :start-date (js/Date. (:start-date %))
```

Comment:
```suggestion
            games (mapv #(assoc % :start-date (js/Date. (:start-date %))
```
`map` is lazy and we wanna make sure we "realize" the whole collection before inserting it into the state atom.